recipes/vasp/v6.5.0/mc/environments.yaml

```diff
@@ -0,0 +1,33 @@
+vasp:
+  mpi:
+    spec: cray-mpich@9.0.0
```

Comment:
I think `9.0.0` is a pre-release. Do we want to use it in what we provide to the users?

Reply:
I just took the latest version from the cluster config. It's now reverted back to 8.1.32.